core/src/core-api/navigation.js

```diff
@@ -48,6 +48,19 @@ class LuigiNavigationManager {
     return new linkManager().navigate(path, preserveView, modalSettings, splitViewSettings, drawerSettings);
   }
 
+  /**
+   * Offers an alternative way of navigating with intents. This involves specifying a semanticSlug and an object containing parameters.
+   * @memberof LuigiNavigation
+   * @param {string} semanticSlug concatenation of semantic object and action connected with a dash (-)
+   * @param {Object} params an object representing all the parameters passed (optional, default '{}')
+   * @example
```

Comment:
```suggestion
   * @since NEXTRELEASE
   * @example
```

Also should this be noted here as NEXTRELEASE and then the docu script will then add it automtically to docs/luigi-core-api.md